noggin/utility/__init__.py

```diff
@@ -60,10 +60,11 @@ def fn(*args, **kwargs):
 
 
 def group_or_404(ipa, groupname):
-    try:
-        return ipa.group_show(groupname)
-    except python_freeipa.exceptions.NotFound:
+    group = ipa.group_find(groupname)['result']
+    if ipa.group_find(groupname)['result'] == []:
         abort(404)
```

Comment:
Maybe add a message, like `abort(404, f"Group {groupname} could not be found.")`